src/test/kotlin/com/example/pass_manager/data/TestDataFixture.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.pass_manager.data
+
+import com.example.pass_manager.domain.MongoClient
+import com.example.pass_manager.domain.MongoPass
+import com.example.pass_manager.domain.MongoPassType
+import org.bson.types.ObjectId
+import java.math.BigDecimal
+import java.time.Instant
+
+class TestDataFixture {
```

Comment:
Maybe change from 'class' to 'object', because you need only one instance of this class